### DISCUSSION TITLE
Cloud seams at the extremities

### DISCUSSION THREAD

**Discussion by BastienChapuis** [matteason/live-cloud-maps#1] [General]
Hi matteason !

First of all ; thanks for your update ! 

For the subject, there is seams in the Pacific zone and the image don't loop. I don't know if you can do something about that or if we need to apply an offset or something like that for mask it ?

**Comment by matteason** [comment]
Thanks for raising this - I'll convert to an issue and comment more over there

**Comment by matteason** [comment]
Logged as https://github.com/matteason/live-cloud-maps/issues/2, thanks for reporting @BastienChapuis !

**Comment by matteason** [comment]
@BastienChapuis This should now be fixed, please see https://github.com/matteason/live-cloud-maps/issues/2#issuecomment-1822724095. Let me know if it's working for you.

**Reply by BastienChapuis** [reply]
I confirm ! No more seam, thank you ! :)
